doc/source/api/solver/fieldinfo.rst

```diff
@@ -0,0 +1,98 @@
+.. _ref_field_info:
+
+Field info
+==========
+
+You can use field info objects to access Fluent field information.
+
+Accessing field info objects
+----------------------------
+
+In order to access field info, launch the fluent solver, and make field
+data available (for example, by reading case and data files):
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case-dats", file_name=mixing_elbow_case_path)
+  >>> solver.solution.initialization.hybrid_initialize()
+
+
+The field info object is an attribute of the solver object:
+
+.. code-block:: python
+
+  >>> field_info = solver.field_info
+
+Sample requests
+---------------
+
+Here are the methods for requesting field information:
+
+- ``get_fields_info`` for getting fields information.
+- ``get_range`` for getting the range of the field.
+- ``get_vector_fields_info`` for getting vector fields information.
+- ``get_surfaces_info`` for getting the surfaces information.
+
+Get fields info
+~~~~~~~~~~~~~~~
+You can request the fields information (field name, domain, and section) by
+calling the ``get_fields_info`` method.
+
+.. code-block:: python
+
+  >>> field_info.get_fields_info()
+  {'pressure': {'display_name': 'Static Pressure', 'section': 'Pressure...', 'domain': 'mixture'},
```

Comment:
We really need this interface to return a list of `Field` objects with the contained data rather than a raw data structure (i.e. not a dictionary).  Maybe note that in the issue about improving this API.

Reply:
@dnwillia-work and @seanpearsonuk, I have created a new issue as pre the requirements in this comment. #1285

Reply:
Thanks for adding that, @prmukherj.